### ISSUE TITLE
What's the hyperparameters for CIFAR100 and ImageNet?

### ISSUE THREAD

**Issue by ziyunli-2023** [sgvaze/generalized-category-discovery#9]
Hi, I check the bash script， and there only has scars' hyperparameters？
Are all the datasets using the same hyperparameters?

Thanks!

**Comment by sgvaze** [comment]
Hi! All models are trained with the same hyper-parameters defined by default in the bash scripts. This could be sub-optimal but made for easier reproducibility.

**Comment by kleinzcy** [comment]
@sgvaze Maybe, the patch size has to be tuned for different datasets. The unified patch size is not suitable for images whose size is 32x32. But, in my opinion, there is not necessary to train VIT on CIFAR100.

**Comment by sgvaze** [comment]
For all datasets, we need to resize and normalize to ImageNet hyper-params (including 224 x 224 image size) because we do not fine-tune the entire DINO backbone. 

Fine-tuning the entire backbone for large image sizes requires a lot of memory with large batch sizes and/or contrastive training.
